app/src/main/java/com/vn/kienphung/music_52/ui/home/HomeFragment.java

```diff
@@ -0,0 +1,26 @@
+package com.vn.kienphung.music_52.ui.home;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.vn.kienphung.music_52.R;
+
+public class HomeFragment extends Fragment {
+
+    public static HomeFragment newInstance() {
+        return new HomeFragment();
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View viewRoot = inflater.inflate(R.layout.fragment_home, container, false);
+        return viewRoot;
```

Comment:
```suggestion
        return view;
```
Tương tự ở Fragment khác